static/css/style.css

```diff
@@ -10,7 +10,6 @@
     --s-pink: #FF5A60;
     --s-teal: #23BBBB;
     --t-grey: #E0E7FF;
-    --t-orange: #E8461030;
```

Comment:
Why was the tertiary orange color?

Reply:
Hi @TravelTimN I am not sure why was this added or removed. When I pulled last upstream from master, I had a merging conflict on style.css with two very different files. As advised by Stefan, I accepted incoming change and made sure the only change made by @Ri-Dearg before me was saved.Then I committed and pushed.